flask_peewee/rest/__init__.py

```diff
@@ -770,6 +794,34 @@ def delete(self, obj):
         res = obj.delete_instance(recursive=self.delete_recursive)
         return self.response({'deleted': res})
 
+    def json_edit(self, obj, field_name, path):
+        data = self.read_request_data()
+        value = getattr(obj, field_name)
+        for key in path:
+            value = value.setdefault(key, {})
+
+        value.update(data)
+        obj.save()
+        return self.response(self.serialize_object(obj))
```

Comment:
I missed this the first time around, but I find it surprising that the response is the entire object rather than just what is specified in the field_name + path. Please help me understand the reasoning behind that.